doc-content/enterprise-only/project-data/container-alias-con.adoc

```diff
@@ -0,0 +1,43 @@
+[id='container-alias-con_{context}']
+= Container alias
+
+Container alias is a proxy to the KIE server instances and helps in handling different versions of the same server deployment. It is possible to link a single alias to different versions of a container, and when the container is upgraded, the linked alias automatically points to the new version of the container.
```

Comment:
```suggestion
Container alias is a proxy to the KIE server instances and helps in handling different versions of the same server deployment. It is possible to link a single alias to different versions of a container, and for instance, when the container is upgraded, the linked alias automatically points to the new version of the container.
```

Reply:
or for example. is just an example of how it could be used.

Reply:
done